components/UseImageDialog.vue

```diff
@@ -145,6 +145,10 @@ export default {
         };
       });
     },
+    likeCount() {
+      const { like } = this.image;
+      return like ? like.count : 0;
```

Comment:
So many `like ? like.count`, I think we should assuming it exist as image property and enforce it at API object creation level?

Reply:
Updated

Reply:
There is one left in `components/TheLandingCarousel.vue`.